test/test_training_log.py

```diff
@@ -46,6 +46,7 @@ def test_training_log(self, path='test_training_log.log'):
             automl_settings["log_file_name"] = None
             automl_experiment.fit(X_train=X_train, y_train=y_train,
                                   **automl_settings)
+            automl_experiment._selected.update(None, 0)
```

Comment:
This line is added intentionally to test the update function of SearchState? Does not look good.

Reply:
It's for improving the coverage. Any alternative way to improve coverage?

Reply:
Remember this is testing code.